pkg/openstack/loadbalancer_test.go

```diff
@@ -534,3 +535,32 @@ func Test_getListenerProtocol(t *testing.T) {
 		})
 	}
 }
+
+func TestLbaasV2_updateServiceAnnotations(t *testing.T) {
+	service := &corev1.Service{
+		ObjectMeta: v1.ObjectMeta{
+			Annotations: nil,
+		},
+	}
+
+	annotations := map[string]string{
+		"key1": "value1",
+		"key2": "value2",
+	}
+
+	lbaas := LbaasV2{}
+	lbaas.updateServiceAnnotations(service, annotations)
+
+	serviceAnnotations := make([]map[string]string, 0)
+	for key, value := range service.ObjectMeta.Annotations {
+		serviceAnnotations = append(serviceAnnotations, map[string]string{key: value})
+	}
+
+	expectedAnnotations := []map[string]string{
+		{"key1": "value1"},
+		{"key2": "value2"},
+	}
+
+	assert.Len(t, serviceAnnotations, len(expectedAnnotations))
```

Comment:
`ElementsMatch()` checks the length too, I think this is redundant.

Reply:
noted and changed !
i wanted to be sure i am catching all error :|